Documentation/amdgpu_plugin.txt

```diff
@@ -35,6 +35,82 @@ Dependencies
     This work is rebased on latest criu release available at this time.
 
 
+OPTIONS
+-------
+Optional parameters can be passed in as environment variables before
+executing criu command.
+
+*KFD_DESTINATION_GPUS*::
+    Override GPUs on local restore node. GPUs can be specified as gpu_id
+    (hex or decimal) or minor number. Default:<None>
+
+    E.g:
+
+    KFD_DESTINATION_GPUS=0xff31,0x90db
+
+    KFD_DESTINATION_GPUS=65329,37083
+
+    KFD_DESTINATION_GPUS=renderD129,renderD128
+
+
+*KFD_TOPOLOGY_CHECK*::
+    Enable or disable topology check.
+    If enabled, amggpu_plugin will check whether the destination GPU has
+    sufficient capabilities to run checkpointed application. This option is
+    only valid if KFD_DESTINATION_GPUS is used. Default:Enabled
```

Comment:
Remove the comment about KFD_DESTINATION_GPUS. If this depends on KFD_DESTINATION _GPUS, should you remove KFD_TOPOLOGY_CHECK as well?

Reply:
ACK - correct. KFD_TOPOLOGY_CHECK is removed as well